l10n_it_fatturapa_out/security/ir.model.access.csv

```diff
@@ -1,3 +1,5 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_fatturapa_attachment_out,access_fatturapa_attachment_out,model_fatturapa_attachment_out,account.group_account_invoice,1,1,1,1
 access_wizard_export_fatturapa,access_wizard_export_fatturapa,model_wizard_export_fatturapa,account.group_account_invoice,1,1,1,1
+ir_actions_account_invoice,ir_actions_account_invoice,base.model_ir_actions_actions,account.group_account_invoice,1,0,0,0
```

Comment:
Per coerenza con le altre ACL e quelle Odoo standard (vedi ad esempio https://github.com/odoo/odoo/blob/16.0/addons/account/security/ir.model.access.csv) puoi usare il prefisso `access_`? In questo caso specifico lo userei sia per la colonna `id` sia per la colonna `name`.
Per il resto LGTM :+1:

Reply:
Cambiato!

Reply:
:+1: